vita3k/config/src/config.cpp

```diff
@@ -118,12 +118,12 @@ ExitCode serialize_config(Config &cfg, const fs::path &output_path) {
 ExitCode init_config(Config &cfg, int argc, char **argv, const Root &root_paths) {
     // Always generate the default configuration file
     Config command_line{};
-    serialize_config(command_line, root_paths.get_base_path() / "data/config/default.yml");
-    // Load base path configuration by default; otherwise, move the default to the base path
-    if (fs::exists(check_path(root_paths.get_base_path())))
-        parse(cfg, root_paths.get_base_path(), root_paths.get_base_path());
+    serialize_config(command_line, root_paths.get_pref_path() / "data/config/default.yml");
+    // Load config path configuration by default; otherwise, move the default to the pref path
+    if (fs::exists(check_path(root_paths.get_config_path())))
+        parse(cfg, root_paths.get_config_path(), root_paths.get_pref_path());
     else
-        fs::copy(root_paths.get_base_path() / "data/config/default.yml", root_paths.get_base_path() / "config.yml");
+        fs::copy(root_paths.get_pref_path() / "data/config/default.yml", root_paths.get_config_path() / "config.yml");
```

Comment:
here is so wrong, data folder, is inside base path only
i have explain, never using pref path for any thing

Reply:
You are right. This file should go to cache_path instead of data/config/

Reply:
Fixed.